taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -5,46 +5,22 @@
 loader: taskgraph.loader.transform:loader
 
 transforms:
-    - ffios_taskgraph.transforms.screenshots:transforms
+    - ffios_taskgraph.transforms.bitrise:transforms
     - ffios_taskgraph.transforms.secrets:transforms
     - taskgraph.transforms.job:transforms
     - taskgraph.transforms.task:transforms
 
+kind-dependencies:
+    - build
 
 job-defaults:
+    bitrise-workflow: jlorenzo_L10nScreenshotsTests
+    build-derived-data-path: {artifact-reference: '<build/public/build/target.zip>'}
     description: Generate localized screenshots by delegating the work to bitrise.io
-    run:
-        using: run-commands
```

Comment:
Common configuration has been move to the `transforms` file so that both `build` and `generate-screenshots` can have shared values.